cmd/kubeless/update.go

```diff
@@ -53,7 +53,23 @@ var updateCmd = &cobra.Command{
 			logrus.Fatal(err)
 		}
 
-		err = utils.UpdateK8sCustomResource(runtime, handler, file, funcName, ns)
+		description, err := cmd.Flags().GetString("description")
+		if err != nil {
+			logrus.Fatal(err)
+		}
+
+		labels, err := cmd.Flags().GetStringSlice("label")
+		if err != nil {
+			logrus.Fatal(err)
+		}
+
+		envs, err := cmd.Flags().GetStringSlice("env")
+		if err != nil {
+			logrus.Fatal(err)
+		}
+
+		funcType := "HTTP"
+		err = utils.UpdateK8sCustomResource(runtime, handler, file, funcName, funcType, ns, description, labels, envs)
```

Comment:
See suggestion elsewhere about just building+passing a `Function` object here rather than expanding the arguments to take all possible parameters for a Function.